dashboard-plugin/FederationDashboardPlugin.js

```diff
@@ -560,40 +565,35 @@ class NextMedusaPlugin {
       nextjs: true
     }).apply(compiler);
 
-    return;
+    const hostData = path.join(
+      compiler.options.output.path,
+      compiler.options.name + "-" + filename
+    );
     compiler.hooks.done.tap(PLUGIN_NAME, () => {
-      const sidecarData = path.join(
-        compiler.options.output.path,
-        `child-dashboard.json`
-      );
-      const hostData = path.join(
-        compiler.options.output.path,
-        "dashboard.json"
-      );
-      console.log("sidecar data", sidecarData);
-      console.log("host data", hostData);
-      if (fs.existsSync(sidecarData) && fs.existsSync(hostData)) {
-        console.log("will write merged files");
+      if (fs.existsSync(hostData)) {
         fs.writeFileSync(
           hostData,
-          JSON.stringify(mergeGraphs(require(sidecarData), require(hostData)))
+          JSON.stringify(require(hostData))
         );
       }
     });
 
-    compiler.hooks.afterDone.tap("NextMedusaPlugin", (stats, done) => {
-      if (fs.existsSync(sidecarData) && fs.existsSync(hostData)) {
+    compiler.hooks.afterDone.tap("NextMedusaPlugin", (stats) => {
+      if (fs.existsSync(hostData)) {
         const dashboardData = fs.readFileSync(hostData, "utf8");
-        MedusaPlugin.postDashboardData(dashboardData)
-          .then(done)
-          .catch(done);
-      } else {
-        done();
+        this.postDashboardData(dashboardData)
+          .then(() => {
+            console.info('Data has been successfully sent to the dashboard')
+          })
+          .catch((error) => {
+            console.error('Failed to send data to the dashboard:', error)
+          });
       }
     });
+
   }
 }
 
 module.exports = FederationDashboardPlugin;
 module.exports.clientVersion = require("./client-version");
-module.exports.NextMedusaPlugin = NextMedusaPlugin;
+module.exports.NextMedusaPlugin = NextMedusaPlugin;
```

Comment:
@brunos3d this should most likely export as NextMedusaPlugin and NextZephyrPlugin. We should also have deprecation warning on the medusa plugin.